manual/parts/threads.tex

```diff
@@ -253,7 +253,10 @@ \subsection{Scheduling Context Donation}
 a notification object with a passive thread blocked waiting on it, the passive thread will receive
 the scheduling context that is bound to the notification object.  The scheduling context is returned
 when the thread blocks on the notification object.  This feature allows for passive servers to use
-notification binding (See \autoref{sec:notification-binding}).
+notification binding (See \autoref{sec:notification-binding}).  If a scheduling context is bound to
+both a notification object and a thread, the behaviour will be the same as for a passive server:
```

Comment:
```suggestion
both a notification object and a thread, the behaviour will be the same as for a passive server.
```

Reply:
I think the `:` is better. What follows is an explanation for the preceding sentence.

Reply:
To me ':' seemed more appropriate because the following sentence is just a summary of previously described behaviour for passive tasks and not new information, but I'll change to whatever you prefer.